pkg/cmd/search/search.go

```diff
@@ -51,8 +51,8 @@ func NewSearchCmd(f *cmdutil.Factory) *cobra.Command {
 			# Search for objects in the "BOOKS" index matching the query "tolkien" while setting the number of hits per page and specifying the page to retrieve
 			$ algolia search BOOKS --query "tolkien" --hitsPerPage 2 --page 4
 
-			# Search for objects in the "BOOKS" index matching the query "tolkien" and export the results to a new line delimited JSON (ndjson) file
-			$ algolia search BOOKS --query "tolkien" > books.ndjson
+			# Search for objects in the "BOOKS" index matching the query "tolkien" and export the results to a .json file
+			$ algolia search BOOKS --query "tolkien" > books.json
```

Comment:
Could we showcase how to use the `-o=jsonpath` output in order to export **only** the results to the `books.json` ?

Example: `-o="jsonpath={$.Hits}"`

WDYT?

Reply:
Yes I agree, I tried to write something easier to understand, wdyt?

Reply:
Looks good 👍